### DISCUSSION TITLE
MRTK3 - missing examples

### DISCUSSION THREAD

**Discussion by gelsogrove** [microsoft/MixedRealityToolkit-Unity#11801] [General]
Hello, i can't see all the examples after check all the mrtk3 checkbox inside the MixedRealityFeatureToo
for istance i can't see the MRTK UX Component > Slates as i see on the video tutorial
it weird that not all of them are there isn't it ? any idea ?

![image](https://github.com/microsoft/MixedRealityToolkit-Unity/assets/15196278/66c667dd-615f-4300-8c73-aad1aacd1639)

**Comment by gustav71** [comment]
The same with me...any suggestions?

**Reply by gustav71** [reply]
A rough solution, but fast and works for me- just check all under MRTK3 in MixedRealityFeatureTool.
